sunpy/map/compositemap.py

```diff
@@ -3,18 +3,35 @@
 Author: `Keith Hughitt <keith.hughitt@nasa.gov>`
 """
 import matplotlib.pyplot as plt
+from matplotlib.image import AxesImage, _ImageBase
+from matplotlib.contour import ContourSet, QuadContourSet
+from matplotlib.collections import QuadMesh, Collection
 
 import astropy.units as u
 
 from sunpy.map import GenericMap
-from sunpy.util import expand_list
+from sunpy.util import expand_list, get_keywords, get_set_methods
 from sunpy.visualization import axis_labels_from_ctype, peek_show, wcsaxes_compat
 
 __all__ = ['CompositeMap']
 
 __author__ = "Keith Hughitt"
 __email__ = "keith.hughitt@nasa.gov"
 
+# Valid keyword arguments for each plotting method
+VALID_IMSHOW_KWARGS = ACCEPTED_IMSHOW_KWARGS = get_keywords(
```

Comment:
Do we need both valid and accepted?

Reply:
No, only the `ACCEPTED_*_KWARGS` are used. Just added both for verbosity, as for example `VALID_PCOLORMESH_KWARGS` and `ACCEPTED_PCOLORMESH_KWARGS` are different. Maybe the `VALID_*_KWARGS` variables shouldn't be created as they're neither used nor documented.

Reply:
If we can get away with only one of them, I would prefer that.